themes/kiali/theme.toml

```diff
@@ -0,0 +1,21 @@
+# theme.toml template for a Hugo theme
+# See https://github.com/gohugoio/hugoThemes#themetoml for an example
+
+name = "Kiali"
+license = "MIT"
+licenselink = "https://github.com/yourname/yourtheme/blob/master/LICENSE.md"
+description = ""
+homepage = "http://example.com/"
```

Comment:
Probably needs more work.

Reply:
Yes, this is the scaffolded file. I haven't worked on this file.

Reply:
Fixed